pkg/secrets/check_rights_windows.go

```diff
@@ -24,7 +24,7 @@ var (
 
 // checkRights check that the given filename has access controls set only for
 // Administrator, Local System and the datadog user.
-func checkRights(filename string) error {
+func checkRights(filename string, options checkRightOptions) error {
```

Comment:
Let's make it clear options are not used for the windows version:
```suggestion
func checkRights(filename string, _ checkRightOptions) error {
```